volumes.xconf

```diff
@@ -14,9 +14,13 @@
             </analyzer>
             <text qname="tei:div">
                 <field name="within" expression="if (@type = ('section', 'document')) then 'documents' else ()"/>
+                <field name="volume-id" expression="if (@type = ('section', 'document')) then  root(.)//tei:idno[@type='frus'] else ()"/>
```

Comment:
The canonical location for the FRUS volume ID is `root(.)/tei:TEI/@xml:id` - an expression that might even perform a bit better too.

Reply:
Actually, in hsg-facets, I used `<facet dimension="volume-id" expression="ancestor::tei:TEI/@xml:id"/>`.